### ISSUE TITLE
Incorrect voice over when tapping on the `X` button in the onboarding

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-ios#11995]
## Steps to reproduce
1. Launch Firefox and reach the onboarding.
2. Tap on the `X` button and listen.
### Expected behavior
The close button should be pronounced instead of the Button sheet close, button.
### Actual behavior
Button sheet close, button can be heard instead of only close button.
### Device & build information
* Device: iPhone 13 Pro max (15.1.1).
* Build version: 106 (19336).
### Notes
Attachments:

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4980)

**Comment by data-sync-user** [comment]
➤ Daniela Arcese commented:

We already have a string for this - check why its not used

**Comment by TwizzyIndy** [comment]
I've only found that we've got `accessibilityIdentifier` for `IntroViewController`'s `closeButton` and there is no specific localized string for it.

https://github.com/mozilla-mobile/firefox-ios/blob/f52642d52b1e92477b6b7237c151835d864d3afa/Client/Frontend/Intro/IntroViewController.swift#L25-L30

https://github.com/mozilla-mobile/firefox-ios/blob/f52642d52b1e92477b6b7237c151835d864d3afa/Client/Application/AccessibilityIdentifiers.swift#L135-L149

https://github.com/mozilla-mobile/firefox-ios/blob/f52642d52b1e92477b6b7237c151835d864d3afa/Client/Frontend/Strings.swift#L592-L691
